.github/workflows/validate-licenses.yaml

```diff
@@ -19,12 +19,20 @@ jobs:
           ref: ${{ github.head_ref }}
           fetch-depth: 0
           token: ${{ secrets.GITHUB_TOKEN }}
-      - name: Setup asdf
-        uses: asdf-vm/actions/setup@v2
+
+      - name: Install NIX
+        uses: cachix/install-nix-action@v24
+
+      - name: Install devbox
+        uses: jetpack-io/devbox-install-action@v0.7.0
+        with:
+          enable-cache: true
+          skip-nix-installation: true
+
       - name: Generate image list
         run: |
           export PATH=${PWD}/.local/bin:$PATH
-          make install-tool.helm install-tool.go.envsubst install-tool.go.gojq
+          make install-tool.go.envsubst install-tool.go.gojq
```

Comment:
These can also be installed via devbox btw, although `envsubst` is a different project it provides the same usage.